pkgs/tools/networking/phodav/default.nix

```diff
@@ -28,6 +41,6 @@ in stdenv.mkDerivation rec {
     homepage = "https://wiki.gnome.org/phodav";
     license = licenses.lgpl21;
     maintainers = with maintainers; [ ];
```

Comment:
Maybe consider adding yourself as maintainer here or the other packages

Reply:
Done.